garrysmod/lua/vgui/dhorizontaldivider.lua

```diff
@@ -145,8 +145,7 @@ function PANEL:StartGrab()
 
 	self:SetCursor( "sizewe" )
 
-	local x, y = self.m_DragBar:CursorPos()
-	self:SetHoldPos( x )
+	self:SetHoldPos( self.m_DragBar:CursorPos() )

```

Comment:
Stuff like this could lead to confusion and errors, especially should `SetHoldPos` get a second argument.

Reply:
Just wrap the CursorPos function call in parentheses to remove the second arg.

Reply:
Good idea, sorted with fe3b0ed7b07d91a8e591d19a084087fdb705196f